dashboard/app/views/levels/show.html.haml

```diff
@@ -22,6 +22,9 @@
   :javascript
     var appOptions = #{app_options.to_json};
   = render partial: 'levels/unplug', locals: {app: @game.app}
+- elsif @level.is_a?(Applab) && app_options['readonlyWorkspace'] && !app_options[:channel]
+  -# A teacher is trying to view a student's progress on a level they have not started.
+  = render partial: 'levels/not_started'
```

Comment:
Laurel, you are correct, I added the partial because I thought we were going to want to apply some styling to that page, based on the element ID. However, Sarah has given the OK for how this looks as-is, so we may not need any such styling any time soon. Based on that I'm inclined to go with Josh's suggestion with your follow-up comments and do the minimum which is needed for now, which is to put the text inline and avoid the partial. I'm open to being persuaded either way, though.